src/components/InputValidationMessage/InputValidationMessage.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Used by form inputs such as TextInput, to display a validation message for an invalid input.
+ */
+
+const InputValidationMessage = ({
+  children,
+}) => (
+  <div className="font-color-danger font-size-sm" style={{ marginTop: 'var(--size-spacing-xs)' }}>{children}</div>
```

Comment:
we might want to allow for consumers to pass a className prop directly to this component. In cases where a input helper may be using base font color, or just any use case that requires overriding the default.

Reply:
since the current thinking is we don't intend for consumers to directly use this component (just like the FormLabel), I'm going to leave off a className prop for now. If we find we need it later, we can add it then.